include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -396,7 +396,11 @@ class CommonOperatorLoader : public ProtobufLoader {
                       "Can't register more than outputs in the operation.");
     numOutputs = (numOutputs < 0) ? op.output_size() : numOutputs;
     for (int i = 0; i < numOutputs; i++) {
-      nodeValueByName_[op.output(i)] = NodeValue(node, i);
+      // If output name is specified to run in FP16 precision and output type
+      // is fp16 add convert to fp32 node and update nodeValueByName_ map,
```

Comment:
nit: this is a little stale due to reference to `nodeValueByName_ ` which is no longer there -- can you at least say something like e.g. "update nodeValueByName_ via the call to [...]"

Reply:
Done